sh/drive-utils.sh

```diff
@@ -151,8 +173,12 @@ _generate_upload_link() {
         -D - || :)" && _clear_line 1 1>&2
     _clear_line 1 1>&2
 
-    uploadlink_upload_file="$(printf "%s\n" "${uploadlink_upload_file##*[L,l]ocation: }" | while read -r line; do printf "%s\n" "${line%%$(printf '\r')}" && break; done)"
-    { [ -n "${uploadlink_upload_file}" ] && return 0; } || return 1
+    case "${uploadlink}" in
```

Comment:
Codacy found an issue: [uploadlink is referenced but not assigned.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6363604)